include/mbedtls/ssl.h

```diff
@@ -1188,6 +1188,11 @@ struct mbedtls_ssl_session {
     unsigned char MBEDTLS_PRIVATE(mfl_code);     /*!< MaxFragmentLength negotiated by peer */
 #endif /* MBEDTLS_SSL_MAX_FRAGMENT_LENGTH */
 
+/*!< RecordSizeLimit received by peer */
```

Comment:
Otherwise, see comment line 1176 in this file, we miss some changes in test code: in ssl_populate_session() and ssl_serialize_session_save_load(). What about ssl_session_copy()?